AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -123,6 +123,28 @@ typedef struct {
  */
 + (void)setRangeModeForMemoryWarnings:(ASLayoutRangeMode)rangeMode;
 
+/**
```

Comment:
Maybe add a note of some kind here - "Rasterization is a powerful optimization feature, but can sometimes have unpredictable effects.  We have this option in +Beta to reflect the fact that some rarer edge cases may cause issues.  Be sure to test any configurations you use Rasterization with."